man/mono.1

```diff
@@ -1160,18 +1160,18 @@ For platforms that do not otherwise have a way of obtaining random bytes
 this can be set to the name of a file system socket on which an egd or
 prngd daemon is listening.
 .TP
-\fBMONO_ENABLE_COOP\fR
-This makes the Mono runtime and the SGen garbage collector run in cooperative
-mode as opposed to run on preemptive mode.   Preemptive mode is the mode
+\fBMONO_DISABLE_COOP\fR
```

Comment:
My feeling is that we should preserve both environment variables, in case things change in the future